IFC4x3/Properties/p/Packers_3Nc5dkhQPFdAnZR5V5geX1/DocProperty.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Packers_3Nc5dkhQPFdAnZR5V5geX1" Name="Packers" UniqueId="d79859ee-ada6-4f9c-ac63-6c57c5aa8841" PropertyType="p_enumeratedvalue" PrimaryDataType="IfcLabel">
```

Comment:
Rename to "InjectionPackerType"?

Reply:
Sounds better to me. Ping @JontyDunn and @hessf-ilf

Reply:
Agreed, sounds better.

Reply:
Name changed to InjectionPackerType.